src/olympia/constants/promoted.py

```diff
@@ -159,6 +159,7 @@ def __bool__(self):
     name=_('Partner'),
     api_name=PROMOTED_GROUP_CHOICES.PARTNER.api_value,
     high_profile=True,
+    high_profile_rating=True,
```

Comment:
Aren't these constants obsolete? Do we need to be keep them up to date any longer?  (shouldn't they be deleted all together? - not in this PR though)

Reply:
Yeah, they will be -- this is just to prevent one of the sync tests from failing in the mean time.